src/Config.js

```diff
@@ -5,7 +5,7 @@ class Config {
     Config.miraHostName = process.env.MIRA_HOSTNAME || (process.env.SESSION_SERVICE_CONTAINERIZED && process.env.SESSION_SERVICE_CONTAINERIZED.toLowerCase() === 'true' ? 'mira' : 'localhost');
     Config.miraPort = 9100;
 
-    Config.sessionStrategy = process.env.SESSION_STRATEGY || 'leastload';
+    Config.sessionStrategy = process.env.SESSION_STRATEGY || 'roundrobin';
```

Comment:
I'm not sure round robin is better as default, I think we should adjust the least-loaded algorithm instead :)